infra/src/test/kotlin/fi/fta/geoviite/infra/geometry/GeometryAssertions.kt

```diff
@@ -0,0 +1,91 @@
+import fi.fta.geoviite.infra.geometry.*
+import fi.fta.geoviite.infra.tracklayout.TrackLayoutTrackNumber
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+
+fun assertPlansMatch(original: GeometryPlan, planFromDb: GeometryPlan) {
```

Comment:
Siirretty toisesta IT-testistä omaan tiedostoon jotta voi käyttää eri testiluokista